src/model/device_state.rs

```diff
@@ -12,6 +13,7 @@ use super::ve_bus_state::VeBusState;
 pub enum DeviceState {
     TestRecord(TestRecordState),
     SolarCharger(SolarChargerState),
+    GridCharger(GridChargerState),
```

Comment:
Please rename this and associated items to AcCharger as per the Victron documentation.